app/Components/SignUp/index.js

```diff
@@ -3,46 +3,47 @@ import { View, StyleSheet, Text, ScrollView, Switch } from 'react-native';
 import { SafeAreaView } from 'react-native'
 import { Colors, Typography } from '_styles';
+import i18next from 'i18next';
 
 
 const SignUp = ({ }) => {
     return (
         <SafeAreaView style={styles.mainContainer}>
             <View style={styles.container}>
                 <ScrollView style={{ flex: 1 }} showsVerticalScrollIndicator={false}>
-                    <Header headingText={'Sign Up'} subHeadingText={'Please confirm your details.'} />
+                    <Header headingText={i18next.t('label.signup')} subHeadingText={i18next.t('label.signup_confirm')} />
                     <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
-                        <Input label={'FIRST NAME'} value={'Paulina'} />
-                        <Input label={'LAST NAME'} value={'Sanchez'} style={{ marginLeft: 15 }} />
+                        <Input label={i18next.t('label.firstname')} value={'Paulina'} />
+                        <Input label={i18next.t('label.lastname')} value={'Sanchez'} style={{ marginLeft: 15 }} />
                     </View>
-                    <Input label={'EMAIL'} value={'startplanting@trees.com'} />
+                    <Input label={i18next.t('label.email')} value={'startplanting@trees.com'} />
                     <View style={styles.selectRoleBtnsContainer}>
                         <View style={[styles.roleBtnContainer, styles.marginRight]}>
-                            <Text style={styles.roleText}>Individual</Text>
+                            <Text style={styles.roleText}>{i18next.t('label.individual')}</Text>
                         </View>
                         <View style={[styles.roleBtnContainer, styles.marginLeft]}>
-                            <Text style={styles.roleText}>Company</Text>
+                            <Text style={styles.roleText}>{i18next.t('label.company')}</Text>
                         </View>
                     </View>
                     <View style={styles.selectRoleBtnsContainer}>
                         <View style={[[styles.roleBtnContainer, styles.activeRoleContainer], styles.justifyCenter, styles.marginRight]}>
-                            <Text style={[styles.roleText, styles.primaryText]}>Tree Planting Organisation</Text>
+                            <Text style={[styles.roleText, styles.primaryText]}>{i18next.t('label.tpo_title')}</Text>
                         </View>
                         <View style={[styles.roleBtnContainer, styles.marginLeft]}>
-                            <Text style={styles.roleText}>School</Text>
+                            <Text style={styles.roleText}>{i18next.t('label.education_title')}</Text>
                         </View>
                     </View>
-                    <Input label={'NAME OF TREE PLANTING ORGANIZATION'} value={'Forest in Africa'} />
+                    <Input label={i18next.t('label.tpo_title_organisation')} value={'Forest in Africa'} />
```

Comment:
This should be dynamic `Name of {{roleText eg: company, Organization, School, TPO}}`

Reply:
will the user select or input the roleText?